src/survey/scss.coffee

```diff
@@ -0,0 +1,25 @@
+{go, map, tee, reject,
+include, Type, isType, isMatch,
+Method,
+glob} = require "fairmont"
+
+{define, context, sass} = require "panda-9000"
```

Comment:
TODO: Change `sass` to `scss` when P9K https://github.com/pandastrike/panda-9000/issues/15 is fixed and deployed to NPM